templates/fine-tune-llama2/README.md

```diff
@@ -0,0 +1,36 @@
+# Fine-tuning Llama-2 models with Deepspeed, Accelerate, and Ray Train
+This template shows you how to fine-tune Llama-2 models. 
+
+## Step 1: Kick off the fine-tuning
+- Open a terminal in VS Code (ctl+shift+`). 
+- Choose the model size you want (7b, 13b, or 70b) with the `--size` option
+- Run command below to kick off fine-tuning with dummy data, Grade School Math 8k (GSM8K) dataset.
+
+```
+# The flag `--as-test` is for demo / testing purposes as it runs through only one forward and backward pass of the model and checkpoints the model. It will take 6-7 mins. Without this flag, it will take ~42 mins (3 epochs).
+
+python train.py --size=7b --as-test
+```
+
+Model checkpoints will be stored under `{user's first name}/ft_llms_with_deepspeed/` in the cloud stroage bucket created for your Anyscale account. The full path will be printed in the output after the training is completed.
+
+
+## Step 2: Test the fine-tuned model
+
+
```

Comment:
Something missing? 🐳

Reply:
Yeah.. This is a placeholder. I'm still trying to figure out how to load the checkpoint locally and run it.

Reply:
I removed this step for now. Need your help to write the code for this step. Resolve it for now.